sensorpush_to_statusdb.py

```diff
@@ -378,8 +378,8 @@ def process_data(sensors_json, samples_dict, start_time, nr_samples_requested):
 
         # The dropna is needed since sometimes we get sparse samples
         # and might have hours without samples.
-        hourly_mean = sensor_samples.resample("1H").mean().dropna()
-        for hour, mean_val in hourly_mean.iteritems():
+        hourly_mean = sensor_samples.resample("1h").mean().dropna()
```

Comment:
Did it warn about this as well?

Reply:
Yeah `FutureWarning: 'H' is deprecated and will be removed in a future version, please use 'h' instead.
  hourly_mean = sensor_samples.resample("1H").mean().dropna()`